libs/dh/wholesale/feature-overview/.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Doesn't look like something we like to need (and maintain) in every single feature folder.